src/native/managed/cdacreader/src/Data/MethodTable.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Diagnostics.DataContractReader.Data;
+
+internal sealed class MethodTable : IData<MethodTable>
+{
+    static MethodTable IData<MethodTable>.Create(Target target, TargetPointer address) => new MethodTable(target, address);
+    public MethodTable(Target target, TargetPointer address)
+    {
+        Target.TypeInfo type = target.GetTypeInfo(DataType.MethodTable);
+
+        MTFlags = target.Read<uint>(address + (ulong)type.Fields[nameof(MTFlags)].Offset);
+        BaseSize = target.Read<uint>(address + (ulong)type.Fields[nameof(BaseSize)].Offset);
+        MTFlags2 = target.Read<uint>(address + (ulong)type.Fields[nameof(MTFlags2)].Offset);
+        EEClassOrCanonMT = target.ReadPointer(address + (ulong)type.Fields[nameof(EEClassOrCanonMT)].Offset);
+        Module = target.ReadPointer(address + (ulong)type.Fields[nameof(Module)].Offset);
+        ParentMethodTable = target.ReadPointer(address + (ulong)type.Fields[nameof(ParentMethodTable)].Offset);
+        NumInterfaces = target.Read<ushort>(address + (ulong)type.Fields[nameof(NumInterfaces)].Offset);
+        NumVirtuals = target.Read<ushort>(address + (ulong)type.Fields[nameof(NumVirtuals)].Offset);
+    }
+
+    public uint MTFlags { get; init; }
```

Comment:
Is the `MTFlags` name adding clarity here? It is on the `MethodTable` table so I'd imagine `MT` would be implied. Is there some other nuance I am missing?

Reply:
Just "Flags" was annoying to search for. I'm not strongly attached to the name, though.

Reply:
I see. Yes, that makes sense to me.

Reply:
> I see. Yes, that makes sense to me.

Feel free to keep this as is.